src/App/CleanupProcess.h

```diff
@@ -25,7 +25,6 @@
 #define APP_CLEANUPPROCESS_H
 
 #include <functional>
-#include <FCGlobal.h>
```

Comment:
Do not remove this header!

It's essential for code checkers like clang-tidy or IDEs like QtCreator as otherwise it causes parsing errors. The problem is that these tools assume AppExport is the class name and thus don't know what to do with CleanupProcess.